core/src/apps/solana/transaction/programs.json

```diff
@@ -0,0 +1,3270 @@
+{
+  "programs": [
```

Comment:
a very nice thing to have here would be a generator of a text file in very roughly the following format:

```
## System Program

Instruction: Create Account
1. Create Account:  (account) new_account
2. Deposit:         (lamports) lamports
3. From:            (account) funding_account
(not shown)         (u64) space
(not shown)         (authority) owner
```

so that (a) this description is auto-generated, CI checked, and (b) we have a straightforward ability to check which properties are shown and which aren't

i won't hold _this_ PR on it but please add it soon